newIDE/app/src/EventsSheet/EnumerateEventsMetadata.js

```diff
@@ -23,6 +23,7 @@ export const enumerateEventsMetadata = (): Array<EventMetadata> => {
       return extensionEvents
         .keys()
         .toJSArray()
+        .filter(type => type !== 'BuiltinCommonInstructions::Async')
```

Comment:
Why exclude Async from this function?

Reply:
Because it is used for code generation purposes, but not usable as an actual event.